From: Roger Shimizu <rogershimizu@gmail.com>
Date: Sat, 19 Dec 2015 23:25:28 +0900
Subject: change classpath for proguard_gradle

Signed-off-by: Roger Shimizu <rogershimizu@gmail.com>
---
 build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index 237d30e..9d06f93 100644
--- a/build.gradle
+++ b/build.gradle
@@ -109,7 +109,7 @@ subprojects {
                    commons_cli: 'commons-cli:commons-cli:1.2',
                    jflex: 'de.jflex:jflex:1.4.3',
                    jflex_plugin: 'co.tomlee.gradle.plugins:gradle-jflex-plugin:0.0.2',
-                   proguard_gradle: 'net.sf.proguard:proguard-gradle:5.2.1',
+                   proguard_gradle: 'proguard:gradle:5.2.1',
                    dx: 'com.google.android.tools:dx:1.7'
          ]
     }
